storage/interceptors/unique_binding_names_iterceptor.go

```diff
@@ -62,21 +64,32 @@ func (c *uniqueBindingNameInterceptor) AroundTxCreate(h storage.InterceptCreateA
 }
 
 func (c *uniqueBindingNameInterceptor) checkUniqueName(ctx context.Context, binding *types.ServiceBinding) error {
+	operation, operationFound := operations.GetFromContext(ctx)
+	if !operationFound {
+		log.C(ctx).Debug("operation missing from context")
+	}
+
+	if operationFound && operation.Reschedule {
+		log.C(ctx).Debug("skipping unique check of binding name as this is a rescheduled operation")
```

Comment:
Better log it on info level, as we don't expect this to happen too often. Same for the other interceptor

Reply:
addressed